addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -315,6 +315,21 @@
                                             </div>
                                         </div>
                                     </div>
+                                    <div class="col-12 mb-0 py-2 s_website_form_field" style="display: none;">
+                                        <div class="row s_col_no_resize s_col_no_bgcolor">
+                                            <label class="col-4 col-sm-auto s_website_form_label" style="width: 200px" for="recruitment9">
+                                                <span class="s_website_form_label_content">Job Position</span>
+                                                <span class="s_website_form_mark"> *</span>
+                                            </label>
+                                            <div class="col-sm">
+                                                <input id="recruitment9" type="text"
+                                                    class="form-control s_website_form_input"
+                                                    name="job_name" required=""
+                                                    data-fill-with="job_name"/>
+                                                <div class="alert alert-warning mt-2" id="job-message" style="display:none;"></div>
+                                            </div>
+                                        </div>
+                                    </div>
```

Comment:
Does this need all of this stuff? Couldn't it just be the input element wrapped in an invisible div? I'm genuinely asking, I didn't try it out myself.

Reply:
I think so because as soon as I try to remove some of the classes or divs, we lose the functionality.

Reply:
Yes it does indeed seem like studio relies on most of the structure of the xml. It would be possible to strip it down a little but that doesn't really give any benefits.